superset/models/helpers.py

```diff
@@ -278,6 +278,12 @@ def alter_params(self, **kwargs):
         d.update(kwargs)
         self.params = json.dumps(d)
 
+    def remove_params(self, param, **kwargs):
+        d = self.params_dict
+        d.pop(param, None)
+        d.update(kwargs)
```

Comment:
used `kwargs` here.

Reply:
sorry, what I meant was - is this function ever called with keyword arguments? It looks like you only called it with the positional argument `param_to_remove`, so I don't thiink we need the kwargs logic.

Reply:
fixed. Thanks!